business_requirement_deliverable_resource_template/models/business_requirement_resource_template.py

```diff
@@ -0,0 +1,83 @@
+# -*- coding: utf-8 -*-
+# © 2017 Elico Corp (https://www.elico-corp.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models
+from openerp.exceptions import ValidationError
+from openerp.tools.translate import _
+
+
+class BusinessRequirementDeliverable(models.Model):
+    _inherit = "business.requirement.deliverable"
+
+    def _prepare_resource_lines(self):
+        rl_data = self.product_id.sudo().resource_lines.copy_data()
+        return [(0, 0, item) for index, item in enumerate(rl_data)]
```

Comment:
@dreispt How to do it on the equivalent number of lines?
With copy_data() I have a dictionary like:
```
[{'name': u'Odoo developer',
  'product_id': 60,
  'qty': 1.0,
  'resource_type': u'task',
  'uom_id': 5},
 {'name': u'Odoo Functional Consultant',
  'product_id': 61,
  'qty': 1.0,
  'resource_type': u'task',
  'uom_id': 5}]
```

then on the return the values prepared are like this:

```
[(0,
  0,
  {'name': u'Odoo developer',
   'product_id': 60,
   'qty': 1.0,
   'resource_type': u'task',
   'uom_id': 5}),
 (0,
  0,
  {'name': u'Odoo Functional Consultant',
   'product_id': 61,
   'qty': 1.0,
   'resource_type': u'task',
   'uom_id': 5})]
```

and this structure of data suit perfect on assignment:

```
self.resource_ids = self._prepare_resource_lines()
```